transport-native-unix-common/src/main/c/netty_unix_socket.c

```diff
@@ -229,7 +229,7 @@ static jint _socket(JNIEnv* env, jclass clazz, int type) {
     return fd;
 }
 
-int netty_unix_socket_initSockaddr(JNIEnv* env, jbyteArray address, jint scopeId, jint jport,
+int __attribute__((visibility("hidden"))) netty_unix_socket_initSockaddr(JNIEnv* env, jbyteArray address, jint scopeId, jint jport,
```

Comment:
consider grouping all the "exported" methods together in the file, and surrounding with a "Exported JNI methods ..." comment

Reply:
Actually these are not even exported JNI methods. They are just exported c methods within the library

Reply:
From the perspective of a native library I would say "the symbols are not exported, but you can always refer to them via function pointers". In this context we are referring to the function pointers to register/export the symbols with JNI.